server/lib/save_data_and_file.coffee

```diff
@@ -20,6 +20,11 @@ module.exports = (log, model, options, tags) ->
 
         # For each entry...
         async.eachSeries entriesToSave, (entry, callback) ->
+            if (entry.date not instanceof moment)
+                log.error 'Moment instance expected for date field'
+                log.info 'Bill creation aborted'
+                return callback()
```

Comment:
What if I remove the log.error and then return the same message in the callback ?